### ISSUE TITLE
The FAQ is for everyone

### ISSUE THREAD

**Issue by alexhultman** [uNetworking/uWebSockets.js#131]
Dearest @orefalo,

I ran your k6 and this is what I came up with:

1. k6 debug feature is nonsensical, broken and inaccurate - it does not show correct headers & thus you provided invalid info. I told you the headers you provided made no sense, but you know better, of course.

2. k6 in default settings work just like it should, with no issues, just like any modern web browser do. No difference here.

3. Your "issue" when doing no-reuse is entirely and exactly what I told you from the start, 114, entirely so. uWS does not actively close connections unless timed out, it assumes a modern client.

Basically, stop and listen once in a while, mate.

Sincerest
